azerothcore/azerothcore-wotlk
# fix(db): Updated Durotar shaman quests

## Changes Proposed:
- Missing RewardNextQuest for 'Call of Earth'
- Made 'Earth Sapta' a repeatable quest to mimic the blue question marks
  from retail (Classic/Vanilla).

## Issues Addressed:

- Closes not reported bug.

## SOURCE:

Sniffs and Youtube

Link on the moment the blue question mark from '_Earth Sapta_' becomes available.
https://youtu.be/nyntgWJHf1s?t=120

## Tests Performed:

- Tested in-game

## How to Test the Changes:


Must include this #9683 to test the Blue question mark
1. Create a Shaman Troll/Orc
2. .Level 3
3. Do the Shaman quest line and check for the blue question mark and the open next quests when finish another.

## Known Issues and TODO List:


None.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
